packages/kit/src/runtime/client/router.js

```diff
@@ -88,7 +88,7 @@ export class Router {
 					'sveltekit:scroll': scroll_state()
 				};
 				history.replaceState(new_state, document.title, window.location.href);
-			}, 50);
+			}, 200);
```

Comment:
should we add a comment here like:
```suggestion
			// iOS scroll event intervals happens between 30-150ms, sometimes around 200ms
			}, 200);
```